### PR TITLE
fix(external docs): Update GitHub org name throughout the website

### PR DESCRIPTION
**Pull request by lucperkins**
I definitely strived to not rename anything that shouldn't but there may be places I went awry. So reviewers, please do go over this pretty fastidiously. There are some remaining `timberio` references for things like Docker images, as we're not ready to make that switch yet.

### CODE REVIEW

File: website/content/en/docs/setup/installation/package-managers/homebrew.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ weight: 4
 ## Installation
 
 ```shell
-brew tap timberio/brew && brew install vector
+brew tap vectordotdev/brew && brew install vector
```
Comment (spencergilbert) [inline website/content/en/docs/setup/installation/package-managers/homebrew.md:12]: I wonder if this was actually changed?

File: website/content/en/docs/setup/installation/platforms/kubernetes.md [hunk 2]
```diff
@@ -50,12 +50,12 @@ namespace: vector
...
   # Include Vector recommended base (from git).
-  - github.com/timberio/vector/distribution/kubernetes/vector-agent?ref=v{{< version >}}
+  - github.com/vectordotdev/vector/distribution/kubernetes/vector-agent?ref=v{{< version >}}
 
 images:
   # Override the Vector image to avoid use of the sliding tag.
-  - name: timberio/vector
-    newName: timberio/vector
+  - name: vectordotdev/vector
+    newName: vectordotdev/vector
```
Comment (spencergilbert) [inline website/content/en/docs/setup/installation/platforms/kubernetes.md:58]:
```suggestion
  - name: timberio/vector
    newName: timberio/vector
```
These refer to the docker repo

**Review by spencergilbert — CHANGES_REQUESTED** [review]

File: website/content/en/docs/setup/installation/package-managers/homebrew.md (hunk 1 above)
Comment (jszwedko) [reply website/content/en/docs/setup/installation/package-managers/homebrew.md:12]: Yeah, this should be updated. `timberio/brew` still works due to Github redirecting it.

File: website/content/en/docs/setup/installation/platforms/kubernetes.md (hunk 2 above)
Comment (jszwedko) [reply website/content/en/docs/setup/installation/platforms/kubernetes.md:58]: 👍 just for reference, renaming the docker org is mentioned in https://github.com/vectordotdev/vector/issues/9137

File: website/content/en/docs/setup/installation/package-managers/homebrew.md (hunk 1 above)
Comment (spencergilbert) [reply website/content/en/docs/setup/installation/package-managers/homebrew.md:12]: Cool, not familiar enough with brew taps to know exactly how that works 😆

**Review by jszwedko — COMMENTED** [review]
Thanks for this clean-up @lucperkins ! I echo Spencer's comment about not changing the docker image name, but otherwise the rest look good to me.

**Review by jszwedko — APPROVED** [review]
🚀

**Pull request merged**